container/test.bzl

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A wrapper around container structure tests for Bazel.
+
+This rule feeds a built image and a set of config files
+to the container structure test framework."
+"""
+
+load("//container:flatten.bzl", "container_flatten")
+
+def _impl(ctx):
+    flat_image = ctx.attr.image
+
+    config_runfiles = []
+    config_str = ""
+
+    for config_file in ctx.files.configs:
+        config_runfiles.append(config_file)
+        config_str = config_str + '$(pwd)/' + config_file.short_path + ' '
+
+    image_name = "bazel/%s:%s" % (ctx.attr.image.label.package, ctx.attr.image.label.name)
```

Comment:
I'm not sure if there's a better mechanism for getting the loaded images name here, but having one would be nice. Maybe the image loader could output something?

Reply:
Users can override the `bazel/` portion. :(

Reply:
A trick: feed the image into `container_bundle` and name it w/e you want

Reply:
good idea, I'll give that a shot